tcf_website/views/review.py

```diff
@@ -3,9 +3,13 @@
 
 from django import forms
 from django.contrib.auth.decorators import login_required
+from django.contrib.auth.mixins import LoginRequiredMixin # For class-based views
```

Comment:
Did you run `./precommit`? This goes against [PEP 8](https://www.python.org/dev/peps/pep-0008/#inline-comments), which says:
> Inline comments should be separated by at least two spaces from the statement.

Reply:
nope I don't think I did rip, but why does it still pass pylint?

Reply:
Yeah `pylint` doesn't check everything `autopep8` checks

Reply:
fixed